client/plots/regression.js

```diff
@@ -61,7 +61,8 @@ class Regression {
 			termfilter: appState.termfilter,
 			config,
 			allowedTermTypes: appState.termdbConfig.allowedTermTypes,
-			minTimeSinceDx: appState.termdbConfig.minTimeSinceDx
+			minTimeSinceDx: appState.termdbConfig.minTimeSinceDx,
+			hideRegressionTables: appState.termdbConfig.hideRegressionTables
```

Comment:
hideRegressionTables is not user configurable so it shouldn't be in state. it bloats state